### PR TITLE
Update the client specification support list. 

### PR DESCRIPTION
**Pull request by SadieCat**
- Update client spec support
- Remove/replace some dead clients

### CODE REVIEW

**Comment by ProgVal** [comment]
Could you use a tag or commit id in links, instead of `master`? Links to `master` may break because files or code is moved.

**Comment by SadieCat** [comment]
Tag comments are kinda useless for referencing because when you go to update the list it just points to an old commit and you have to find the branch anyway.

**Comment by ProgVal** [comment]
but it's useful to provide a source for the existing data. And if someone wants to see the latest version, they can replace with `master` in the URL

File: _data/sw_clients.yml [hunk 1]
```diff
@@ -397,6 +430,7 @@
           - plain
     - name: The Lounge
       # ref: https://github.com/thelounge/thelounge/projects/1
+      #      https://github.com/kiwiirc/irc-framework/blob/master/docs/ircv3.md
```
Comment (ProgVal) [inline _data/sw_clients.yml:432]: wrong link

Comment (SadieCat) [reply _data/sw_clients.yml:432]: It's not the wrong link. The Lounge uses irc-framework.

Comment (ProgVal) [reply _data/sw_clients.yml:432]: ok, my bad

**Comment by DanielOaks** [comment]
Thanks for the PR, much appreciated! <3

**Pull request merged**